sdk/python/feast/infra/offline_stores/snowflake.py

```diff
@@ -91,6 +91,9 @@ class SnowflakeOfflineStoreConfig(FeastConfigBaseModel):
     schema_: Optional[str] = Field(None, alias="schema")
     """ Snowflake schema name """
 
+    authenticator_: Optional[str] = Field("", alias="authenticator")
```

Comment:
Was "authenticator" a reserved word?

Reply:
No, it's not a reserved keyword. I'll make the adjustment.